app/lib/three_scale/redis_config.rb

```diff
@@ -9,12 +9,6 @@ def initialize(redis_config = {})
 
       @config = ActiveSupport::OrderedOptions.new.merge(raw_config)
       config.sentinels = parse_sentinels(sentinels) if sentinels
-
-      # The ID is forced to be nil to disable the default behavior in Sidekiq < 6
-      # which invokes CLIENT SETNAME command, which incompatible with some Redis providers
-      # see https://issues.redhat.com/browse/THREESCALE-9210
-      # This can be removed when Sidekiq is upgraded to a version >= 6.0.7
-      config.id = nil
```

Comment:
I guess this is to be merged after sidekiq 7 is  merged.

Reply:
No, because we're already using Sidekiq `6.4.2`, and any version `>= 6.0.7` already includes the fix.